academy/static/src/js/demo_rpc.js

```diff
@@ -0,0 +1,53 @@
+odoo.define('academy.demo_rpc', function (require) {
+    'use strict';
+
+    var core = require('web.core');
+    var Widget = require('web.Widget');
+    var rpc = require('web.rpc');

```

Comment:
![image](https://github.com/AntonioMartinD/Building_a_website/assets/35231827/facc5b4e-3761-45ab-865e-430383f7f97c)


since odoo 140. You can use rpc in this way